### DISCUSSION TITLE
Modification Power and temperature

### DISCUSSION THREAD

**Discussion by tomtomtac** [philibertc/micronova_controller#30] [Q&A]
Hello,
I try to integrate the modification of the power, and the temperature setpoint. If I send {0x80, 0x58, 0x50, 028} to increase power, {0x80, 0x58, 0x54, 0x2C} to decrease power; {0x80, 0x58, 0x52, 0x2A} to increase the setpoint, or {0x80, 0x58, 0x58, 0x30} to decrease the setpoint, the stove screen is activated, but neither the set point nor the power changes.... I must be missing something! An idea? Thank you

**Comment by philibertc** [comment]
Hello,

You have to write these parameters on the EEPROM so 0xA0 and not 0x80. On the other hand I did not implement this functionality voluntarily: in case of a bug, for example something that runs in a loop, there is a risk of writing many times on the EEPROM which is a memory with a limited number of writes (about 10,000) and therefore of breaking this memory. So first I have to find a way to secure all this.

What you can do in the meantime is to connect an IR led on one of the free GPIOs, and to adapt the program to implement this (I could help you if needed).

Sincerely,
philibertc

**Reply by tomtomtac** [reply]
Hello,

Thank you ! I will try with 0xA0 to do some tests, by I had understood that it was possible to simulate the action of the remote control as described in comments #68 and #70 on K3a.me..... me English being very bad I probably misunderstood!! In any case, thank you very much, great project, congratulations!!

**Comment by wes1993** [comment]
@philibertc Hello, but if we use the remote we also write to eeprom is correct? (the remote send the command to MOBO that write the eeprom)

**Reply by philibertc** [reply]
Hello, Yes, exactly, but this way we can be sure that the memory is managed in a good way and that there are no eventual bugs. Anyway, these are not settings to be changed very often.

Best Regards,
philibertc

**Comment by wes1993** [comment]
Yes thanks a lot Philibert!! Perfect :)
